apidoc/Titanium/UI/iPhone/ListViewSeparatorStyle.yml

```diff
@@ -7,6 +7,10 @@ extends: Titanium.Proxy
 since: "3.2.0"
 platforms: [iphone, ipad]
 createable: false
+deprecated:
+    since: "5.2.0"
+    removed: "7.0.0"
+    notes: Use <Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE> and <Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_NONE> instead.
```

Comment:
Rather "Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE" it should be "Titanium.UI.TABLE_VIEW_SEPARATOR_STYLE_SINGLE_LINE". Same with "Titanium.UI.UI.TABLE_VIEW_SEPARATOR_STYLE_NONE".